packages/discord.js/typings/index.d.ts

```diff
@@ -4908,12 +4917,12 @@ export interface ClientEvents {
   stickerUpdate: [oldSticker: Sticker, newSticker: Sticker];
   guildScheduledEventCreate: [guildScheduledEvent: GuildScheduledEvent];
   guildScheduledEventUpdate: [
-    oldGuildScheduledEvent: GuildScheduledEvent | null,
+   oldGuildScheduledEvent: GuildScheduledEvent | PartialGuildScheduledEvent | null,
```

Comment:
```suggestion
    oldGuildScheduledEvent: GuildScheduledEvent | PartialGuildScheduledEvent | null,
    ```
Umm a little bit unfortunate edit